lib/src/models/community_feed.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+import 'package:lightning_api/src/models/models.dart';
+
+part 'community_feed.g.dart';
+
+@JsonSerializable(explicitToJson: true)
+class CommunityFeed extends Equatable {
```

Comment:
Please add a test for serialize / deserialize. This 1) makes it easy to review the code at the API level (important because it is more difficult to change), and 2) can protect against changes which accidentally break the API.

Reply:
Same with other test statement, I'm too exhausted at the moment and would like to add it later on if that suits you

Reply:
I can't give a proper review without seeing what the API looks like. But I understand being too exhausted at the moment to do that. Better to wait than to deploy something when the developer is exhausted.